administration/src/bp-modules/cards/CreateCardsButtonBar.tsx

```diff
@@ -16,7 +15,7 @@ const CreateCardsButtonBar = ({ cardBlueprints, generateCards, goBack }: CreateC
   return (
     <ButtonBar>
       <Button icon='arrow-left' text='Zurück zur Auswahl' onClick={goBack} />
-      <Tooltip2
+      <Tooltip
```

Comment:
why did you switch to tooltip?

Reply:
Actually, I only switched because the tests had problems at first, and I assumed it was related to the deprecated Tooltip2 component. I can change it back in this PR, but at some point we (again) might want to move away from deprecated components (-> #1001 )

Reply:
ok i'm fine. dunno if the disappearing problem also occurs with tooltip2